src/eth/types.ts

```diff
@@ -87,7 +87,6 @@ export const EthErc4337AccountStruct = object({
     enums([
       `${EthMethod.PersonalSign}`,
       `${EthMethod.Sign}`,
-      `${EthMethod.SignTransaction}`,
```

Comment:
Why do we remove this one here?

Reply:
The 4337 account doesn't have the ability to sign a regular transaction. If they support feat/add-isEVMAccount-check they can sign messages

Reply:
Oh so I might have re-introduced that during the split of account types? 😅